src/databricks/labs/ucx/install.py

```diff
@@ -248,21 +248,35 @@ def _create_jobs(self):
         self._create_readme()
         self._create_debug(remote_wheel)
 
+    @staticmethod
+    def _sort_task_list() -> list[Task]:
+        return sorted(_TASKS.values(), key=lambda x: x.task_id)
+
+    @classmethod
+    def _step_list(cls) -> list[str]:
+        step_list = []
+        [step_list.append(task.workflow) for task in cls._sort_task_list() if task.workflow not in step_list]
```

Comment:
can you rewrite it as a for-loop, as you're not assigning a result of list comprehension to a value